### PR TITLE
Add some fixes to readme

### PR DESCRIPTION
**Pull request by alexrqs**
This PR includes:

- How to run watch task
- Add 'bash' suffix for sintax highlight on the readme file

### CODE REVIEW

**Comment by alexrqs** [comment]
related with #131

File: README.md [hunk 1]
```diff
@@ -22,11 +22,19 @@ If you maintain a JavaScript library that has UI components, we'd love for you t
 ### Building
 
 Once you've cloned Chassis to your machine, run these commands:
-```
+```bash
 $ npm install
 $ grunt build
 ```
 
+### Develop
```
Comment (sfrisk) [inline README.md]: Development

File: README.md [hunk 2]
```diff
@@ -22,11 +22,19 @@ If you maintain a JavaScript library that has UI components, we'd love for you t
...
+
+Now that you have the environment ready, you may want to 'watch' your build task, so you can run:
```
Comment (sfrisk) [inline README.md]: Reword along the lines of "To automate the build process during development, run the 'watch' task to compile SCSS upon file change."

Comment (geekman-rohit) [inline README.md]: :+1:  that sounds much much better

Comment (sfrisk) [inline README.md]: Any change this can get updated soon? Other than this fix, this PR is good to be pulled in.

Comment (jzaefferer) [inline README.md]: I think you can just fix this yourself while merging?

**Pull request closed without merging**